CMakeLists.txt

```diff
@@ -368,7 +368,7 @@ include_directories(SYSTEM ${MSGPACK_INCLUDE_DIRS})
 option(PREFER_LUA "Prefer Lua over LuaJIT in the nvim executable." OFF)
 
 if(PREFER_LUA)
-  find_package(Lua REQUIRED)
+  find_package(Lua 5.1 EXACT REQUIRED)
```

Comment:
The unstable PPA failed after this:

```
CMake Error at /usr/share/cmake-2.8/Modules/FindPackageHandleStandardArgs.cmake:108 (message):
  Could NOT find Lua: Found unsuitable version "5.1.5", but required is exact
  version "5.1" (found
  /usr/lib/x86_64-linux-gnu/liblua5.1.so;/usr/lib/x86_64-linux-gnu/libm.so)
Call Stack (most recent call first):
  /usr/share/cmake-2.8/Modules/FindPackageHandleStandardArgs.cmake:313 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindLua.cmake:193 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  CMakeLists.txt:371 (find_package)
```

Reply:
Sigh. Well that makes `EXACT` pretty useless. There doesn't seem to be a way to specify a range.

Thanks for the note.

Reply:
Either that or we're handling version checking incorrectly in FindLua.cmake.